app/workers/fetch_failed_graph_worker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Sidekiq Worker for fetching linked data labels
+class FetchFailedGraphWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 11 # Around 2.5 days of retries
+
+  attr_writer :triplestore
+  attr_accessor :uri, :user
+
+  def perform(uri, user)
```

Comment:
See previous comment about pid/uri. This perform will probably need to be re-written to re-fetch & re-index the specific field, not just fetch and dump to graph cache. Whoops, sorry.